packages/core/src/components/hotkeys/hotkeyParser.ts

```diff
@@ -271,9 +271,56 @@ export const getKeyCombo = (e: KeyboardEvent): IKeyCombo => {
  *
  * Unlike the parseKeyCombo method, this method does NOT convert shifted
  * action keys. So `"@"` will NOT be converted to `["shift", "2"]`).
+ *
+ * The windowOverride parameter is for unit testing only. Do not use it
+ * in feature work.
  */
-export const normalizeKeyCombo = (combo: string): string[] => {
+export const normalizeKeyCombo = (combo: string, windowOverride?: any): string[] => {
     const keys = combo.replace(/\s/g, "").split("+");
-    return keys.map((key) => Aliases[key] != null ? Aliases[key] : key);
+    return keys.map((key) => {
+        const keyName = (Aliases[key] != null)
+            ? Aliases[key]
+            : key;
+        return (keyName === "meta")
+            ? getMetaKeyNameForPlatform(windowOverride)
+            : keyName;
+    });
 };
 /* tslint:enable:no-string-literal */
+
+// source: https://stackoverflow.com/a/19883965/5199574
+const WindowNavigatorPlatforms = {
+    MAC: [
+        "Macintosh",
+        "MacIntel",
+        "MacPPC",
+        "Mac68K",
+    ],
+    WINDOWS: [
+        "OS/2",
+        "Pocket PC",
+        "Windows",
+        "Win16",
+        "Win32",
+        "WinCE",
+    ],
+};
+
+const META_KEY_DEFAULT_NAME = "meta";
```

Comment:
There are key-name strings floating around this whole file, but I wanted to at least create semantically meaningful constants for this new code.